### DISCUSSION TITLE
[Suggestions] Adventure map, abandoned mine, there should be no ghost animation for the abandoned mine

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3725] [General]
In the original game, there is no ghost animation for the abandoned mine.

In original game:

![Abandoned mine H2 01](https://user-images.githubusercontent.com/43583994/96159347-d27b7180-0f14-11eb-9b40-90360052c542.png)

In fheroes2:

![Abandoned mine fh2 01](https://user-images.githubusercontent.com/43583994/96159369-d7402580-0f14-11eb-89cc-b67cf8279d3a.png)

**Edit 2020/10/15.**

Similar issue closed without change: https://github.com/ihhub/fheroes2/issues/332.

Ghost animation is still present in fheroes2.
Why was this issue closed without change?

**Comment by idshibanov** [comment]
Personally I prefer the animated version, it follows the spirit of the game where guarding monsters are seen at the entrance, adds to visual clarity. It's also a great animation that really shows mine is "haunted".

**Comment by Branikolog** [comment]
Totally agreed with @idshibanov. 
Previously closed issue was a solid decision of our developer team.

**Comment by LeHerosInconnu** [comment]
@idshibanov and @Branikolog,

The problem is this animation is used for mines guarded by ghosts using the Haunt spell.
This can mislead the player, for example, when a mine is hidden by the fog of war and it is possible to see part of the ghost animation.
The player may think that the mine could produce the much sought-after resource (because the Haunt spell ghost animation is present), but in fact it is not.

And personally, I don't like the reuse of objects, animations, etc. in a game, it looks unprofessional.

To respect the original game, I think the best would be to let the user choose and offer the ghost animation for the abandoned mine as an option.

**Reply by ihhub** [reply]
I converted this issue into discussion till we find a proper solution which will fit the majority of the team.
